docs/automated-workflows/dagman-advance-functionality.rst

```diff
@@ -14,39 +14,42 @@ easily be used for multiple nodes in a DAG with variance.
 
 :index:`VARS (macro for submit description file)<single: DAGMan; VARS (macro for submit description file)>`
 
-Macro Variables for Nodes
-^^^^^^^^^^^^^^^^^^^^^^^^^
-
 .. sidebar:: Example Diamond DAG with VARS
 
     .. code-block:: condor-dagman
+        :caption: Example Diamond DAG descriptions using VARS command
 
         # File name: diamond.dag
-        JOB  A  A.submit
-        JOB  B  B.submit
-        JOB  C  C.submit
-        JOB  D  D.submit
+        JOB  A  A.sub
+        JOB  B  B.sub
+        JOB  C  C.sub
+        JOB  D  D.sub
         VARS A state="Wisconsin"
         PARENT A CHILD B C
         PARENT B C CHILD D
 
     .. code-block:: condor-submit
+        :caption: Example job description file referencing DAG VARS
 
-        # file name: A.submit
+        # file name: A.sub
         executable = A.exe
         log        = A.log
         arguments  = "$(state)"
         queue
 
-    The above DAG file and ``A.submit`` description will result in the
+    The above DAG file and ``A.sub`` description will result in the
     job evoking the following:
 
     .. code-block:: console
+        :caption: Example job execution command for Node A using DAG VARS
 
         $ A.exe Wisconsin
 
+Macro Variables for Nodes
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
 DAGMan can specify ``key=value`` pairs of information to be used within
-a node job submit description as a referable macro via the **VARS**
+a node's associated submit description as a referable macro via the :dag-cmd:`VARS[Usage]`
 command. This information is defined on a per-node basis using the
 following syntax:
```

Comment:
I know this isn't your work, but the prose here is so overly wordy.  Can we try to tighten it up by removing unneeded (and wrong) words?  e.g. `DAGman can specify` Really?  DAGman isn't doing any "specify"ing, the user (or maybe the submission file is.  How about something here like

```
The :dag-cmd:`VARS[Usage]` dag command in the dag description file defines variables using a ``key=value`` syntax.  These variables can then be used in the job description file.  The complete VARS lines looks like: